build.sbt

```diff
@@ -112,6 +114,23 @@ lazy val scalacheck = crossProject(JSPlatform, JVMPlatform)
 lazy val scalacheckJVM = scalacheck.jvm
 lazy val scalacheckJS  = scalacheck.js
 
+lazy val interopSpecs2 = crossProject(JSPlatform, JVMPlatform)
```

Comment:
I'd call it `specs2`, making the artifact's name `weaver-specs2`, which is consistent with the rest of the interop packages (zio / scalacheck)